### DISCUSSION TITLE
Is the bot only usable via self-hosting?

### DISCUSSION THREAD

**Discussion by Crunchbits** [dominikks/discord-soundboard-bot#21] [Q&A] [answered]
I tried pressing adding the bot via "Add bot to server" in https://bot.floomer.app/ to try it out but I get the following:

![Screenshot_pe (3)](https://github.com/dominikks/discord-soundboard-bot/assets/87384615/5b82ca60-da8d-456e-92e0-cfacc3499445)

I just want confirmation whether it's possible or not to add it without self-hosting?

**Comment by dominikks** [comment]
Hey, I meant for it to be publicly available via my instance, at least to try it out. I'll look into why it is not public.

I believe Discord started requiring verification for public bots at some point, so it might have something to do with that.

**Reply by dominikks** [reply, marked as answer]
@Crunchbits I believe the bot should be public now. Feel free to try it out!

**Reply by Crunchbits** [reply]
Yes, it is working now! Thank you.
